### DISCUSSION TITLE
Unable to reverify affiliation for a month

### DISCUSSION THREAD

**Discussion by msperryucsd** [community/community#108428] [GitHub Education]
### Hi everyone,

I have been trying to reverify my academic verification for several weeks but the github education site seems to be stuck. Whenever I try to upload a new image and click "process my application", nothing seems to happen. When I try to check my previous applications, I am unable to and am instead redirected to upload a new image which doesn't work. It also does not say any application is under review. It has been a month of this and I am not sure if I am missing something.

**Comment by DrDallas** [comment]
I could not get to that point using Microsoft Edge. I changed to Chrome and it worked. Once I got to the upload verification area, I tried a PNG and it did not work until I uploaded a JPG instead.

**Comment by DeandrePetersen** [comment]
Sometimes, stored data can interfere with website functionality. Clearing your cache and cookies may help resolve any issues. It's possible that the problem is specific to the browser or device you're using. Try accessing the site from a different browser or device to see if that resolves the issue. As well, sometimes websites undergo maintenance or updates that can temporarily affect functionality. Check if GitHub has posted any announcements about maintenance or known issues on their website or social media channels.
